charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -95,6 +91,36 @@ spec:
             - "{{ .Values.spireServer.socketDir }}/{{ .Values.spireServer.socketFile }}"
             - "--shallow"
         {{- end }}
+
+      - name: tornjak-backend
+        image: {{ .Values.tornjak.config.backend.img }}:latest
```

Comment:
maybe we make the tag also a value that we can modify later? perhaps not completely necessary now.